packages/hub/src/app/about/page.tsx

```diff
@@ -0,0 +1,58 @@
+import ReactMarkdown from "react-markdown";
+import { NarrowPageLayout } from "@/components/layout/NarrowPageLayout";
+import remarkGfm from "remark-gfm";
+import {
+  GITHUB_URL,
+  GITHUB_DISCUSSION_URL,
+  DISCORD_URL,
+  GRAPHQL_URL,
+  NEWSLETTER_URL,
+  QURI_DONATE_URL,
+} from "@/lib/common";
+
+const markdown = `
+# About Squiggle Hub
+Squiggle Hub serves as a communal platform for the creation and sharing of code written in [Squiggle](https://www.squiggle-language.com/). Both the platform and the source code are open-source and free to use.
+
+## Squiggle: A Glimpse
+[Squiggle](https://www.squiggle-language.com/) is a probabilistic estimation-oriented programming language that operates on the JavaScript platform. We think it's well-suited for intuition-based modeling and risk estimation. Additional details can be found on the Squiggle official website.
+
+## Our Connection to Guesstimate
+Out team previously co-founded [Guesstimate](https://getguesstimate.com/), a different estimation tool. Squiggle Hub stands as a spiritual successor to Guesstimate. With the power of a full programming language at its disposal, it offers superior flexibility and power, albeit at the cost of a steeper learning curve.
+
+## QURI & Squiggle Hub
+The [Quantified Uncertainty Research Institute](https://quantifieduncertainty.org/), or QURI, is the proud creator of Squiggle Hub. As a registered 501(c)(3) nonprofit, QURI is devoted to the design and research of uncertainty quantification tools. Our ambition is for Squiggle Hub to aid effective nonprofits and funding bodies in making impactful decisions.
+
+## Key Links
+- [Squiggle](https://www.squiggle-language.com/)  
+- [Squiggle API](${GRAPHQL_URL})  
+- [Squiggle Discord](${DISCORD_URL})  
+- [Squiggle Github](${GITHUB_URL})  
+- [Squiggle Github Discussion (For Ideas and Issues)](${GITHUB_DISCUSSION_URL})  
+- [Squiggle Newsletter (Part of the QURI Newsletter)](${NEWSLETTER_URL})
+
+## Licensing
+Squiggle Hub, along with Squiggle, is available for free use, with the code being open-source and licensed under MIT. Access it [here](https://github.com/quantified-uncertainty/squiggle).
```

Comment:
As a reader of this, I'd wonder about the licensing of _models_ on Squiggle Hub here.

Reply:
Yea, I'm also unsure what to do here. Maybe once there are multi-file models, people could add licenses.